README.md

```diff
@@ -285,11 +285,7 @@ background: url("data:image/png;base64,...");
 
 ## Documentation and Resources
 
-- Next.js Documentation - Learn more about the Next.js framework.
-- Sanity Documentation - Explore Sanity’s powerful CMS features.
-- Storybook Documentation - Get started with component-driven development.
-- Sanity Slack Community - Join the community for support and collaboration.
-
-## License and Credits
-
-© Sankitty. Created by Christina Roise.
```

Comment:
In terms of copy right, make sure we have traces of the fact that some of this is built on Sankitty.